### DISCUSSION TITLE
osgearth3.3 load 3dtiles

### DISCUSSION THREAD

**Discussion by XiaobaiSW** [pelicanmapping/osgearth#2388] [Support Q&A]
I used osgearth3.3's osgearth_viewer to load a 3dtiles uri through the .earth file, and the earth file content 
```
<map name="Cesium Ion">
    <ArcGISServerImage name="World Imagery">
        <url>http://services.arcgisonline.com/arcgis/rest/services/World_Imagery/MapServer/</url>
        <nodata_image>http://services.arcgisonline.com/ArcGIS/rest/services/World_Topo_Map/MapServer/tile/100/0/0.jpeg</nodata_image>
    </ArcGISServerImage>
    <ThreeDTiles name="THREED">
		<url>https://tiles.geovisearth.com/base/v1/whitemodel/earth/tileset.json?token=[mytoken]</url>
		</ThreeDTiles>
	<xi:include href="viewpoints.xml"/>
</map>
```
and there is the osgdb_gltf.dll plug-in, the 3dtiles are converted by cesiumlab, and then the packet capture can request the .b3dm data but it cannot be rendered, and there is no information in the log. Is it something I haven't processed, right?
![企业微信截图_17010115854790](https://github.com/gwaldron/osgearth/assets/47914115/5ffe3810-3266-486c-9b19-f66f22d5a3de)
![企业微信截图_17010116261863](https://github.com/gwaldron/osgearth/assets/47914115/3fad22a4-6dcb-409e-b7c5-ae24f5f1258c)

**Comment by gwaldron** [comment]
I encourage you to try the new osgEarthCesium nodekit found in osgEarth 3.5 -- this will be the way moving forward.

Not sure what the issue is with your app; you might try preloading the gtlf/b3dm loader with the command line switch `-l gltf` (or is it `-e gltf` ? I can never remember which.)
